src/transaction/interfaces/ITransactionParameters.ts

```diff
@@ -13,6 +13,8 @@ export interface ITransactionParameters extends ITweakedTransactionData {
     nonWitnessUtxo?: Buffer | undefined;
     estimatedFees?: bigint;
 
+    optionalOutputs?: PsbtOutputExtended[] | undefined;
```

Comment:
remove the | undefined? if this is already optional you dont have to do this.

Reply:
please add Omit<ITransactionParameters, 'optionalOutputs'> to wrap, interaction and unwrap.